snippets/log6x-elasticsearch-unsupported-snip.adoc

```diff
@@ -0,0 +1,14 @@
+// Text snippet included in the following assemblies:
+// observability/logging/cluster-logging-deploying.adoc
+//observability/logging/cluster-logging.adoc
+//
+// Text snippet included in the following modules:
+//
+//
+
+:_mod-docs-content-type: SNIPPET
+
+[NOTE]
+====
+In OpenShift Container Platform (OCP) 4.16, the Elasticsearch Operator is only supported for ServiceMesh, Tracing, and Kiali. This operator will be removed from the OpenShift Operator Catalog in November 2025. The Elasticsearch Operator is no longer supported for log storage, and Kibana is no longer supported in OCP 4.16 and later versions.For more information on the {logging-uc} lifecycle dates, see link:https://access.redhat.com/support/policy/updates/openshift_operators#platform-agnostic[Platform Agnostic Operators].
```

Comment:
```suggestion
In {product-title} 4.16, the Elasticsearch Operator is only supported for ServiceMesh, Tracing, and Kiali. This Operator is planned for removal from the OpenShift Operator Catalog in November 2025. The reason for removal is that the Elasticsearch Operator is no longer supported for log storage, and Kibana is no longer supported in {product-title} 4.16 and later versions. For more information on the {logging-uc} lifecycle dates, see link:https://access.redhat.com/support/policy/updates/openshift_operators#platform-agnostic[Platform Agnostic Operators].
```

Reply:
A few suggestions per https://redhat-documentation.github.io/supplementary-style-guide/#red-hat-openshift-container-platform (OCP is an incorrect form). 

By Operator Catalog do you mean OperatorHub?

Reply:
Oh silly me good catch on the OCP attribute. As far as catalog, I'm just echoing the wording in web console for the Operator so it should be correct.